main/res/values/dimens.xml

```diff
@@ -21,9 +21,9 @@
     <!-- font size dimensions ====================================== -->
 
     <!-- all toolbars except main screen -->
-    <dimen name="textSize_toolbarSingle">20sp</dimen>
-    <dimen name="textSize_toolbarPrimary">18sp</dimen>
-    <dimen name="textSize_toolbarSecondary">16sp</dimen>
+    <!-- (we explicitly use dp for toolbar to avoid cuts offs and follow material guidelines) -->
+    <dimen name="textSize_toolbarSingle">20dp</dimen>
+    <dimen name="textSize_toolbarSecondary">18dp</dimen>
```

Comment:
Is there a reason why you remove the differentiation between a single-line actionbar text and a double-lined?

Reply:
Reason is, that the the textSize_toolbarPrimary was only used for the Cache List but not for the live map, and I wanted to make the text size consistent between different activities. And I didn't found a way to difers between single and multi-line in the actionbar style...

Reply:
Ok, understood. But then we should stay with `textSize_toolbarPrimary` and `textSize_toolbarSecondary` to have a consistent naming for text sizes.

Reply:
Yes, you're right. will do so...

Reply:
Done